### ISSUE TITLE
DEVICE_CLASS_ENERGY was used from tendabeli, this is a deprecated constant

### ISSUE THREAD

**Issue by Alf-Tupper** [JakDoh/tenda_beli_plug_hassint#1]
Logger: homeassistant.const
Source: helpers/deprecation.py:205
First occurred: 29 March 2024 at 23:10:21 (2 occurrences)
Last logged: 29 March 2024 at 23:10:21

DEVICE_CLASS_ENERGY was used from tendabeli, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorDeviceClass.ENERGY instead, please create a bug report at https://github.com/JakDoh/tenda_beli_plug_hassint/issues
DEVICE_CLASS_POWER was used from tendabeli, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorDeviceClass.POWER instead, please create a bug report at https://github.com/JakDoh/tenda_beli_plug_hassint/issues

**Comment by rbaptista1995** [comment]
Yes the solution is edit the sensor.py fille and change to **from homeassistant.const import UnitOfEnergy, UnitOfPower** 
reset home assistant and they work again

I put in attach the fille with the correction
[sensor.zip](https://github.com/user-attachments/files/18376507/sensor.zip)
